lib/util.py

```diff
@@ -77,24 +77,30 @@ def cache_cookies(cache):
         cache.set(CACHE_COOKIES, _cookie_jar.dump())
 
 
-def request(url, headers={}):
+def request(url, headers={}, errors_are_useful=False):
     debug('request: %s' % url)
     req = urllib2.Request(url, headers=headers)
     req.add_header('User-Agent', UA)
-    response = urllib2.urlopen(req)
-    data = response.read()
-    response.close()
+    try:
+        response = urllib2.urlopen(req)
+        data = response.read()
+        response.close()
```

Comment:
if i do so then response is not populated and i get:
UnboundLocalError: local variable 'response' referenced before assignment